src/Neo/SmartContract/Native/NativeContract.cs

```diff
@@ -87,6 +87,11 @@ public abstract class NativeContract
         /// </summary>
         public string Name => GetType().Name;
 
+        /// <summary>
+        /// Active in
+        /// </summary>
+        public virtual Hardfork ActiveIn { get; } = Hardfork.HF_Genesis;
```

Comment:
We can use nullable instead of create HF_Genesis, what do you prefer ?

Reply:
`HF_Genesis` doesn't seem to be needed to me, it can be handled implicitly.

Reply:
You mean ?

```suggestion
        public virtual Hardfork? ActiveIn { get; } = null;
```

Reply:
Fixed